backend/src/server.cpp

```diff
@@ -1,19 +1,19 @@
 #include "server.hpp"
-#include "cryptography.hpp"
+#include "sqlite3.h"
 
 namespace
 {
     std::vector<std::string> encodePasswords(const std::vector<std::string> &passwords)
     {
-        std::vector<std::string> result;
-        result.reserve(passwords.size());
+        std::vector<std::string> breached_passwords;
+        breached_passwords.reserve(passwords.size());
 
         for (const auto &password : passwords)
         {
-            result.push_back(crow::utility::base64encode(password, password.size()));
+            breached_passwords.push_back(crow::utility::base64encode(password, password.size()));
         }
 
-        return result;
+        return breached_passwords;
     }
 }
```

Comment:
**question**: Do we need this helper anymore? The passwords are already encoded in the database.